src/constants/index.tsx

```diff
@@ -101,11 +101,11 @@ export function getChainLabel(chainId: ChainId): string {
  * List of chain IDs that are supported in the current environment: production or development.
  */
 export const ENV_SUPPORTED_CHAIN_IDS =
-  process.env.APP_ENV === 'production' ? [ChainId.MAINNET] : [ChainId.MAINNET, ChainId.GOERLI];
+  process.env.REACT_APP_ENV === 'production' ? [ChainId.MAINNET] : [ChainId.MAINNET, ChainId.GOERLI];
 
 /**
  * Parse through and pick the chains from SUPPORTABLE_WAGMI_CHAINS based on the list from ENV_SUPPORTED_CHAIN_IDS;
  */
-export const SUPPORT_CHAINS_RAINBOW_KIT = process.env.APP_ENV === 'production' ? [mainnet] : [mainnet, goerli];
+export const SUPPORT_CHAINS_RAINBOW_KIT = process.env.REACT_APP_ENV === 'production' ? [mainnet] : [mainnet, goerli];
```

Comment:
Same value, different variables names. Let's remove `SUPPORT_CHAINS_RAINBOW_KIT`. I also have the plan to remove RainbowKit altogether. See the DCA app implementation, it doesn't use Rainbowkit and works perfectly. Much more granular control over the UI.